src/proxy/outbound.rs

```diff
@@ -185,8 +185,8 @@ impl OutboundConnection {
             let inbound_connection_metrics = traffic::ConnectionOpen {
                 reporter: Reporter::destination,
                 derived_source: None,
-                destination: Some(req.source.clone()),
-                source: req.destination_workload.clone(),
+                source: Some(req.source.clone()),
```

Comment:
actually this was intentionally backwards... let me look into it again

Reply:
I noticed your naming and figured you did it intentionally but see our discussion in the issue.
I think that when you pass it from outbound to inbound on same node the source should stay the source.. Only reporter should change.

Reply:
Ah got it